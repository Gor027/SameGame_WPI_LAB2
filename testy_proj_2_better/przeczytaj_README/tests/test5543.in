   T}}TT}TT
T}}  TT}}}TT }} }} }T }}T   
}} T }  }TTT  T}}}T }T}T T}T}  T} TTTTT T TT  }TT}  }} T}T} TTT} }}}}}} }T T T
T}T T }TT }T}}TTT}}    }TTTT T}T }T   TT}} T    T}} T}}T}}T}  }   } }} }
  }T T}  TT  }TT}T   } T   } }TT  }}  } T}}  } T }T}    }T T}T}} 
  }}}} } }T } }}}}}  TT }} }  T}}}   T}  }}}T
TT }T  T T   }}   TT  } }}}  T} }}T }TT}} } TT}T
 }}}T} } }TT   }T} T}}}T}T}} }T}  T 
} TTT
T}}}  T } T TTT}T}  T}   }  }}}}T T  T}T}}}   T TT T}T }}   T}TT
T}TTTT 
