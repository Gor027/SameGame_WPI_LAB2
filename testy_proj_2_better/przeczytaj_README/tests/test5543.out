   T
  }T      T
 }TT}}   TT   } }}
 }}}TT T }T   T}}}       T    }T  T   T
T}}}}}T}}T}  }TTT}   T}}TT T TTT  T   T T T
}T}}TTTT}TTT }TT}TT T}}TT}}T }TT} }  T} }}}      }      }     T
T}TTT}}}}}TT}}}}}}}}}TT}}}T} T}}}}T  T}TT}}T  }TT}  }} T}T  TT}}
T}}}T}}TTTTT}}}}T}}T}}}T}T}}}}T}}}TTTTT}}}}TT}}TT}}}T} T}}}}}T}} } }}
}}TTTTT}}}TTTTT}T}TTT}}T}}}}}}}}T}T}}T}T}}}}TTTTTTTT}T}}}TTTT}TT}}}}}}}}TTT
